app/src/components/border-date/borderDateDirective.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var COLORS = { blue: 'blue', green: 'green' };
+var MLSEC_A_DAY = 86400000;
+
+angular.module('appBorderDateDirective', [])
+    .directive('appBorderColor', function() {
+        return {
+            scope: false,
+            link: function(scope, element, attrs) {
+                var date = scope.course.date.getTime();
+                var now = Date.now();
```

Comment:
I suggest to rename both variables to courseDate(or courseCreationDate) and currentDate

Reply:
OK. I fixed it.